ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION. -->
+
+**I'm submitting a ...** 
+<!--  (check one with "x") -->
+[ ] bug report
+
+[ ] feature request
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4815743)